### ISSUE TITLE
Android app doesn't show pictures from RSS-bot

### ISSUE THREAD

**Issue by Vetgans** [vector-im/riot-android#1477]
The Riot Desktop app shows a preview pic / thumbnail in the posts from the RSS-bot (as long as there is a picture to be shown off course). Clicking it enlarges the picture wonderfully.
The Android app shows the text of the RSS-post as well, but no picture at all. In order to see it you have to leave Riot and click the link to open up another app (webbrowser, feedreader).

Example feed: http://feeds.bbci.co.uk/news/science_and_environment/rss.xml
Not showing pictures is even more of a problem when you create an RSS-feed from (image-)services like Instagram or Flickr (for instance via https://bridge.suumitsu.eu).
Using the RSS-bot like that offers great opportunities to integrate all kinds of services into Riot/Matrix, but not being able to see RSS-bot pictures in the Android app diminishes those oppurtunities again.

It would be great to be able to see the pictures in the Riot Android app itself, just like in the Desktop app.
Therefore it seems related to how the Android app deals with the RSS-bot, and not to the RSS-bot itself. Hence the post in this part of the github.

**Comment by ChrisFernandez** [comment]
I agree I regularly use the desktop app, (and first I think we need a true desktop app, electron plain sucks!) but in the Android app I just download to be able to chat and see the bot news we get in our channel on my way to work I can't see the url previews from the rss feeds :(  please FIX
